pkg/kubelet/dockershim/docker_checkpoint.go

```diff
@@ -34,11 +34,13 @@ const (
 	schemaVersion        = "v1"
 )
 
-type DockershimCheckpoint interface {
+// ContainerCheckpoint provides the interface for process container's checkpoint data
```

Comment:
Can you share a little more about the name `ContainerCheckpoint` and how its the most appropriate replacement for `DockershimCheckpoint`?

Reply:
The golint error here is:
```
docker_checkpoint.go:35:6: type name will be used as dockershim.DockershimCheckpoint by other packages, and that stutters; consider calling this Checkpoint
```

Because there is already a PodSandboxCheckpoint implement this interface. I think ContainerCheckpoint would be more general. May be a DockerCheckpoint, KataCheckpoint or any other type container's Checkpoint will implement this interface in the future.

Reply:
Could `ProcessContainerCheckpoint` be even more specific? Or does that just increase confusion?

Reply:
For this interface name, I think ContainerCheckpoint may be more suitable. There exist an implement of PodSandboxCheckpoint, and the podsandbox is just a special container. But say it's a process container sounds not good ?